### DISCUSSION TITLE
Success stories

### DISCUSSION THREAD

**Discussion by norpol** [tmo1/sms-ie#116] [Show and tell]
Just as an opportunity to show others this project works and a place to say  thank you

**Comment by norpol** [comment]
Thank you for this amazing project, transferring 2.5k SMS and 2 MMS messages  from Android 11 "Google Messaging" to Android 13 GrapheneOS Messaging app worked like a charm. Same with the call history

**Reply by tmo1** [reply]
Thanks so much! I really appreciate the feedback - both the information that the app is working well outside my limited use and test cases, as well as the fact that others find it useful.
